docker-compose.yml

```diff
@@ -7,9 +7,9 @@ services:
       - 3000:80
     volumes:
       - "./data:/app/backend/data"
-    # Optional healthcheck  
+    # Optional healthcheck
     healthcheck:
-      test: "curl -f http://localhost/health"
+      test: "curl -f http://localhost/healthz"
```

Comment:
While this changed to /healthz, which is more standardized, /health continues to work with no plan on being deprecated